### DISCUSSION TITLE
メンターの画像について

### DISCUSSION THREAD

**Discussion by xx-n59** [Futurealize-with-Tech/fwt#47] [General]
メンターの画像は、supabaseとかcloudinaryとかのストレージに上げておく予定？それともフロントに画像はおいておいて、テーブルの方にはパスだけ入れておく予定？どっちを考えているのか知りたいです、時間ある時に返信お願いします

**Comment by Hasuro-R** [comment]
ここ最近GitHubちゃんと見てなくて気付かんかった🙇

画像はCloudinaryでもSupabaseでも何でもいいねんけど、どっかのサービスに保存して、そのurlをデータベースに入れた方が良いと思う！
